wire-grpc-client/src/jvmMain/kotlin/com/squareup/wire/internal/BlockingMessageSource.kt

```diff
@@ -66,7 +66,7 @@ internal class BlockingMessageSource<R : Any>(
         queue.put(Failure(e))
       }
 
-      override fun onResponse(call: Call, response: GrpcResponse) {
+      override fun onResponse(call: Call, response: Response) {
```

Comment:
Previously `okhttp3.Response` and `GrpcResponse` were interchangeable within the JVM implementation. Now they're not a typealias of each other, so we need to explicitly reference the `okhttp3` version.